.gitignore

```diff
@@ -2,6 +2,7 @@ run_project_docker.sh
 get_mongodumps.sh
 node_modules/
 client/node_modules/
+.env.prism_uat.example
```

Comment:
We need to talk about the naming of this and the new approach of spinning up the containers.

I don't think it's efficient for development to go on a confluence page and then try to create a bash script.

Also naming it other than .env is pointless as you don't have any other .env anyways and would ideally remove an execution command (At least in this scenario).